x-pack/plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.test.ts

```diff
@@ -109,6 +109,7 @@ describe('copySavedObjectsToSpaces', () => {
           success: true,
           successCount: filteredObjects.length,
           successResults: [('Some success(es) occurred!' as unknown) as SavedObjectsImportSuccess],
+          warnings: [],
         };
 
         return Promise.resolve(response);
```

Comment:
@elastic/kibana-security I adapted the test for the `copy_to_space` code that consumes the `import` API to take warnings into account, however, I did not adapt the copy to space flyout to display them. 

Do we want that? And if we do, should I do it in this PR, or should it be done by your team in a follow-up, wdyt?

Reply:
> Do we want that?

@elastic/kibana-security will keep me honest here, but I do think we should handle `warnings` in the `copy_to_space` API even though we may not have a valid use case right now. 

> And if we do, should I do it in this PR, or should it be done by your team in a follow-up, wdyt?

I think it makes more sense to keep the scope of your PR smaller, we (Security Team) can handle this new behavior of the API in a follow-up.